libs/telemetry/src/main/java/org/opensearch/telemetry/metrics/HistogramType.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.metrics;
+
+/**
+ * Histogram type, primarily depends on bucketing strategy.
+ */
+public enum HistogramType {
```

Comment:
This should be consumer decision: the API we've discussed had a way to configure buckets

Reply:
removed.